internal/integration/publish_test.go

```diff
@@ -120,25 +129,115 @@ func TestPublish(t *testing.T) {
 		t.Errorf("expected %v to be %v: %#v", got, want, exposures)
 	}
 
-	// Create an export.
-	resp, err = client.Get("/export/create-batches")
+	wait(t, exportPeriod+500*time.Millisecond, "Waiting before export batches")
```

Comment:
This is not retry logic, we have to wait for a certain amount of time to make sure the exposure keys created will fit in an export window. Don't see any major issues with helper method. It might be useful for other tests as well.

Reply:
It's generally not consider good Go style to create a helper that is 1-2 LOC unless their work is incredibly complex. `time.Sleep` is much clearer than `wait`, and it's difficult to know the behavior of `wait` without going an inspecting the function.

Reply:
Done.